offline_folium/__main__.py

```diff
@@ -2,15 +2,20 @@
 import os
 
 import folium
+import folium.plugins
 from .paths import dest_path
 
 def download_all_files():
     if not os.path.exists(dest_path):
         os.makedirs(dest_path)
     for _, js_url in folium.folium._default_js:
         download_url(js_url)
-    for _, js_url in folium.folium._default_css:
```

Comment:
Is there a reason why you've removed the code that processes the default CSS for the main folium package?